docs/01-get-started/05-deployment.md

```diff
@@ -0,0 +1,31 @@
+# Deploying Serverpod
+
+## Introduction
+
+## Server Requirements
+
+Since Serverpod is written in Dart we can compile it to native code and run it on any platform that is supported by the [Dart tooling](https://dart.dev/get-dart#system-requirements).
```

Comment:
```suggestion
Since Serverpod is written in Dart, it can be compiled into native code and runs on any platform that is supported by the [Dart tooling](https://dart.dev/get-dart#system-requirements).
```